### PR TITLE
create functionality with testing complete

### PR DESCRIPTION
**Pull request by Mannyplg**

### CODE REVIEW

File: src/pages/CatNew.js [hunk 1]
```diff
@@ -1,8 +1,93 @@
...
+      onChange={handleChange}
+      value={newCat.url}
+    />
+  </FormGroup>
+</Form>
+
+<Button onClick={handleSubmit} name="submit">
+Submit Cat
+</Button> 
+</div>
```
Comment (elysemontano) [inline src/pages/CatNew.js]: Check indentation for this section

**Review by elysemontano — CHANGES_REQUESTED** [review]
Code looks great, please clean up your indentation on CatNew.js and I will approve

File: src/pages/CatNew.js (hunk 1 above)
Comment (Mannyplg) [reply src/pages/CatNew.js]: Changes nade.

Comment (Mannyplg) [reply src/pages/CatNew.js]: *made...

**Review by elysemontano — APPROVED** [review]

**Pull request merged**